tests/create_release_response.ml

```diff
@@ -0,0 +1,44 @@
+let test_1 =
```

Comment:
How about `gh_v3_api_example` instead of `test_1`?

Reply:
done